service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java

```diff
@@ -121,6 +124,13 @@ public UsersRecord createNewUser(
     newUser.setPrivilegeLevel(PrivilegeLevel.STANDARD);
     newUser.store();
 
+    AuditRecord audit = db.newRecord(AUDIT);
```

Comment:
There are multiple _similar_ chunks of code where these audit records are being created. Would there be a way to create a method somewhere like `createAuditRecord(type, table, ...) -> Audit Record`  that all of the processors would be able to utilize?